ansible/site.yml

```diff
@@ -8,6 +8,7 @@
 ################
 - import_playbook: jumpbox.yml
 - import_playbook: solr.yml
+- import_playbook: fgdc2iso.yml
```

Comment:
:+1: also should be added to the [Makefile](https://github.com/GSA/datagov-deploy/blob/28b155ca534076455967d92daa4567f5a31fb87d/Makefile#L6) for linting.